app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreenModel.kt

```diff
@@ -17,9 +17,6 @@ data class BloodPressureHistoryScreenModel(
     fun create(patientUuid: UUID) = BloodPressureHistoryScreenModel(patientUuid, null, null)
   }
 
-  val hasBloodPressures: Boolean
-    get() = bloodPressures.isNullOrEmpty().not()
```

Comment:
The advantage of having a separate function to check if the model has loaded BPs is that we can change the implementation in one place.

So, if leave this method as is, but change the getter method to `get() = bloodPressures != null`, we can make this fix without needing to change multiple places.

Reply:
Done